src/sentry/snuba/metrics/fields/base.py

```diff
@@ -887,6 +942,17 @@ def generate_bottom_up_derived_metrics_dependencies(
     ) -> Iterable[Tuple[Optional[MetricOperationType], str, str]]:
         return [(None, self.metric_mri, alias)]
 
+    def generate_groupby_statements(
+        self,
+        projects: Sequence[Project],
+        use_case_id: UseCaseKey,
+        alias: str,
+        params: Optional[MetricOperationParams] = None,
+    ) -> List[Function]:
+        raise InvalidParams(
+            f"Metric {get_public_name_from_mri(self.metric_mri)} does not support being grouped by"
```

Comment:
```suggestion
            f"Metric {get_public_name_from_mri(self.metric_mri)} does not support group by"
```
?

Reply:
I will write a different error message because I essentially want to say Cannot groupby this field